### DISCUSSION TITLE
Glitching tp

### DISCUSSION THREAD

**Discussion by zxygabe** [thatskymod/Sky-CotL-Scripts#72] [General]
I've been using canvas and Dukun langit mod for a while now but when I tried today after the update for it, been crashing a whole lot when enabling unlock cosmetics and emotes, and **teleporting** **to** **wax** **collect** has been going crazy, making you teleport or fly everywhere in the map, hopefully it'll be fixed soon:) And a lot more stuff in Dukun isn't working, some buttons, and the running speed part buttons.

**Comment by thatskymod** [comment]
## Resolved in Dukun Langit v0.2.3rc

The issue shall now be revolved as of **Dukun Langit v0.2.3rc**.

The file is available for download down below:

https://github.com/thatskymod/Sky-CotL-Scripts/blob/d021c69403acb5cc6f9d705c510bc3f110e47bb7/Canvas/Libs/libLangit023rc.so

**Note for future readers:**
With the rapid pace at which mods are evolving, it is difficult to keep links on aged discussions up-to-date. If you find that the link is broken, consider searching the repository manually.
